photowall/app/index.html

```diff
@@ -10,7 +10,8 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<!doctype html>
+ <!doctype html>
+<!-- TODO: Fully upgrade to polymer 2.0 elements -->
```

Comment:
Please also file an issue in this repo. You can link to the issue id here and leave it unassigned, unless you have time to pick it up.